src/hotspot/share/runtime/deoptimization.hpp

```diff
@@ -164,8 +165,11 @@ class Deoptimization : AllStatic {
 #endif
 
   private:
-  // Does the actual work for deoptimizing a single frame
-  static void deoptimize_single_frame(JavaThread* thread, frame fr, DeoptReason reason);
+    static void print_ul(CompiledMethod* nm, intptr_t pc, frame& fr, int trap_bci,
+                         const char* reason_name, const char* reason_action);
+
+    // Does the actual work for deoptimizing a single frame
+    static void deoptimize_single_frame(JavaThread* thread, frame fr, DeoptReason reason);
```

Comment:
Please, keep original indent for this lines to match other line around. Actually `private:` has wrong additional space.